starter-code/iteration-2.js

```diff
@@ -0,0 +1,19 @@
+
+function combineBirthdays() {
+
+  return _.chunk(birthdays, 2);
+
+}
+
+var moreBirthdays = ["Lily Evans", "30 January", "James Potter", "27 March",
+                     "Dudley Dursley", "30 June", "Tom Riddle", "31 December"];
+
+function combineAllBirthdays() {
+
+  var allBirthdays = _.concat(_.chunk(moreBirthdays,2), combineBirthdays());
```

Comment:
About the _.chunk(), if you need to do the same thing more than one time, better make a function.